lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -177,6 +177,42 @@ def test_never_update():
     # checks them.
 
 
+@pytest.mark.backend('TkAgg', skip_on_importerror=True)
+@_isolated_tk_test(success_count=1)
+def test_toolbar_button_la_mode_icon():
+    # test that icon in LA mode can be used for buttons
+    # see GH#25164
+    import tempfile
+    import warnings
+
+    from PIL import Image
+
+    import matplotlib
+    import matplotlib.pyplot as plt
+    from matplotlib.backend_tools import ToolToggleBase
+
```

Comment:
These imports should be at the top?

Reply:
sure I was just following the same "coding style" as the other tests in this file.

Reply:
Again, not a problem if thats how the rest of the tests are in this file - definitely an idiosyncratic structure!  OTOH, are we sure we don't want this in `test_tools_backend.py`?

Reply:
sorry I replied in the main thread for this question